packages-content-model/roosterjs-content-model-core/lib/publicApi/table/setTableCellBackgroundColor.ts

```diff
@@ -8,14 +8,17 @@ const DARK_COLORS_LIGHTNESS = 20;
 const BRIGHT_COLORS_LIGHTNESS = 80;
 const White = '#ffffff';
 const Black = '#000000';
+const DEFAULT_COLORS = ['rgb(0,0,0', '#000000', '#ffffff', 'rgb(255, 255, 255)'];
```

Comment:
Why need  this?

It is possible user's default color is not black/white

Reply:
If we are removing the background color, we also need remove this textColor that we added before.

Reply:
Hmm, how can we know if this color is added by the code when we add background color, or added by user's manual action?

Reply:
Maybe check the color override flag from cell metadata. If there is color override flag, background color is added from table cell shade feature, then text color on table cell is also added there. Otherwise, the text color may be from original table. In that case, should we still change the color?

Reply:
And in this change do you also need to clear that flag?

Reply:
To avoid matching text and background colors, I checked if they were equal and also equal to the adaptive text color. I tried metadata approach and it did not work, because it erased the text color when I did these steps: 
  - Add a header row
  - Change the header text color
  - Delete the header row
  - The text color was gone.